<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPaletteWindow</class>
 <widget class="QDialog" name="UserPaletteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="ColorButton" name="color_button_4" native="true"/>
   </item>
   <item row="3" column="2">
    <widget class="ColorButton" name="color_button_14" native="true"/>
   </item>
   <item row="0" column="3">
    <widget class="ColorButton" name="color_button_3" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="ColorButton" name="color_button_8" native="true"/>
   </item>
   <item row="3" column="3">
    <widget class="ColorButton" name="color_button_15" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="ColorButton" name="color_button_5" native="true"/>
   </item>
   <item row="3" column="1">
    <widget class="ColorButton" name="color_button_13" native="true"/>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ColorButton" name="color_button_1" native="true"/>
   </item>
   <item row="2" column="2">
    <widget class="ColorButton" name="color_button_10" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="ColorButton" name="color_button_6" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="ColorButton" name="color_button_9" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="ColorButton" name="color_button_2" native="true"/>
   </item>
   <item row="1" column="3">
    <widget class="ColorButton" name="color_button_7" native="true"/>
   </item>
   <item row="2" column="3">
    <widget class="ColorButton" name="color_button_11" native="true"/>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="red_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="red_out"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="green_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="green_out"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="blue_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="blue_out"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="ColorButton" name="color_button_0" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="ColorButton" name="color_button_12" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QWidget</extends>
   <header>color_button.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserPaletteWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserPaletteWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
